performance.rst

```diff
@@ -106,8 +106,16 @@ make them available to all requests until the server is restarted, improving
 performance significantly.
 
 During container compilation (e.g. when running the ``cache:clear`` command),
-Symfony generates a file called ``preload.php`` in the ``config/`` directory
-with the list of classes to preload.
+Symfony generates a file with the list of classes to preload in the
+``var/cache`` directory.
+
+Rather than use this file directly, when installing with Flex the Framework
+Bundle recipe will create a file called ``preload.php`` in the ``config/``
+directory, which you can use safely, since it includes safeguards in case the
+cache has not been warmed beforehand.
+
+If this file is missing, you can reinstall the recipe by executing
+``composer sync-recipes symfony/framework-bundle --force``.
```

Comment:
`composer recipes:install vendor/package --force -v`, we should recommend this command rather the alias

Reply:
Updated to use the target command instead of its alias.

Reply:
You can keep the package name "symfony/framework-bundle" instead of "vendor/package" 😄

Reply:
Ouch, sorry. Copy and paste + multi tasking do **not** go great together. :sweat_smile: